src/index.html

```diff
@@ -1,13 +1,82 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Stars</title>
     <link rel="stylesheet" href="./style.css">
   </head>
-  <body>
-    <h1>Stars</h1>
+  <body class="body">
+    <ul class="stars stars--0">
+      <li class="stars__star"></li>
+
+      <li class="stars__star"></li>
+
+      <li class="stars__star"></li>
+
+      <li class="stars__star"></li>
+
+      <li class="stars__star"></li>
+    </ul>
+
+    <ul class="stars stars--1">
+      <li class="stars__star stars--yellow"></li>
```

Comment:
you shouldn't use modifier to `stars__star`
get the modifier from stars (stars--1, stars--2 etc) and implement solution 
```suggestion
      <li class="stars__star"></li>
```